core/lensFDHelpers.ml

```diff
@@ -0,0 +1,104 @@
+open Utility
+open LensUtility
```

Comment:
I like having them as submodules in a single Lens module, I'm less happy with putting them all in a single file. A single subdirectory seems nicer, but I'm not entirely sure how OCaml handles subfolders etc?. It is also difficult to have them as a seperate library (like core vs bin), since they depend on `Value.t` and `Types.typ`.

Reply:
You can have your cake and eat it too. Here is how: Create a file `Lens.ml` that contains aliases of the modules Utility, FDHelpers, etc...
```ocaml
module Utility = LensUtility
module FDHelpers = LensFDHelpers
...
```
Create a signature file `Lens.mli` that includes the signatures of the modules Utility, FDHelpers, etc..
```ocaml
module Utility : sig
  include module type of type LensUtility
end

module FDHelpers : sig
  include module type of type LensFDHelpers
end
...
```

Reply:
@dhil we just discussed this in the meeting and decided to defer fine-tuning the organization for now; we can revisit later.